packages/headless/src/api/search/search-metadata.ts

```diff
@@ -6,7 +6,8 @@ export type SearchApiMethod =
   | 'recommendations'
   | 'html'
   | 'productRecommendations'
-  | 'fieldDescriptions';
+  | 'fieldDescriptions'
+  | 'query';
```

Comment:
I don't  think the new value here is necessary, it's not a new Search API method that you are using for folding, you are doing a search query so I think you can just use the value `search` in you request.